numba/cuda/tests/cudadrv/test_runtime.py

```diff
@@ -12,6 +26,42 @@ def test_get_version(self):
                                   (10, 1), (10, 2), (11, 0))
         self.assertIn(runtime.get_version(), supported_versions)
 
+    def test_visible_devices_set_after_import(self):
```

Comment:
This test probably needs to go in it's own class and have the `SerialMixin` mixed in to prevent nested multiprocessing.